University-of-New-England-Australia-Discipline-of-History.csl

```diff
@@ -0,0 +1,611 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>University of New England (Australia) Discipline of History Traditional Footnoting</title>
+    <id>http://csl.mendeley.com/styles/24445091/University-of-New-England-Australia-Discipline-of-History</id>
+    <link href="http://www.zotero.org/styles/University-of-New-England-Australia-Discipline-of-History" rel="self"/>
+    <link href="http://www.zotero.org/styles/universita-di-bologna-lettere" rel="template"/>
+    <link href="http://www.une.edu.au/__data/assets/pdf_file/0007/87811/REF_Trad.-Footnoting.pdf" rel="documentation"/>
+    <author>
+      <name>Benjamin Hollister</name>
+      <email>ben@rootsandbranches.com.au</email>
+      <uri>http://www.mendeley.com/profiles/benjamin-hollister2/</uri>
+    </author>
+    <category citation-format="note"/>
+    <category field="humanities"/>
+    <category field="history"/>
+    <summary>University of New England (Australia) Discipline of History Traditional Footnoting</summary>
+    <updated>2015-09-05T21:56:36+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en-GB">
+    <terms>
+      <term name="page" form="short">
+        <single>p.</single>
+        <multiple>pp.</multiple>
+      </term>
+      <term name="no date">n.d.</term>
+      <term name="ibid" form="short">ibid</term>
+      <term name="ibid" form="long">ibid</term>
+      <term name="cited" form="short">cit.</term>
+      <term name="cited" form="long">op.cit.</term>
+      <term name="issue" form="short">no.</term>
+      <term name="season-01">Spring</term>
+      <term name="season-02">Summer</term>
+      <term name="season-03">Autumn</term>
+      <term name="season-04">Winter</term>
+    </terms>
+  </locale>
+  <macro name="editor-translator">
+    <group delimiter=", ">
+      <choose>
+        <if variable="author">
+          <names variable="editor" delimiter=", ">
+            <name delimiter=", "/>
+            <label form="verb" text-case="lowercase" prefix=" (" suffix=") " strip-periods="true"/>
```

Comment:
no need for all the `text-case="lowercase"` here and further down. Terms are typically lowercased in English, except where they're proper nouns like months. 
Also no need for strip-periods="true" -- long terms like `verb` don't have periods.